azurerm/internal/services/network/express_route_connection_resource.go

```diff
@@ -0,0 +1,340 @@
+package network
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2020-05-01/network"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/tf"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/clients"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/network/parse"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/network/validate"
+	azSchema "github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/tf/schema"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/timeouts"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func resourceArmExpressRouteConnection() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmExpressRouteConnectionCreateUpdate,
+		Read:   resourceArmExpressRouteConnectionRead,
+		Update: resourceArmExpressRouteConnectionCreateUpdate,
+		Delete: resourceArmExpressRouteConnectionDelete,
```

Comment:
Also these four and their definition
```suggestion
		Create: resourceExpressRouteConnectionCreateUpdate,
		Read:   resourceExpressRouteConnectionRead,
		Update: resourceExpressRouteConnectionCreateUpdate,
		Delete: resourceExpressRouteConnectionDelete,
```

Reply:
Updated